src/accounts/AccountPage.tsx

```diff
@@ -132,19 +142,34 @@ const AccountAboutPage: FC = () => {
             <CancellationOverlay onHideOverlay={hideDeactivateAccountOverlay} />
           </Overlay>
         </CancelServiceProvider>
+        {shouldShowDeleteFreeAccountButton && <DeleteFreeAccountButton />}
       </>
     </AccountTabContainer>
   )
 }
 
 const AccountPage: FC = () => {
+  const {account} = useSelector(selectCurrentIdentity)
+
   return (
-    <Page titleTag={pageTitleSuffixer(['Account Settings Page'])}>
-      <AccountHeader testID="account-page--header" />
-      <UsersProvider>
-        <AccountAboutPage />
-      </UsersProvider>
-    </Page>
+    <>
+      <Page titleTag={pageTitleSuffixer(['Account Settings Page'])}>
+        <AccountHeader testID="account-page--header" />
+        <UsersProvider>
+          <AccountAboutPage />
+        </UsersProvider>
+      </Page>
+      <Switch>
+        {CLOUD && account.type === 'free' && (
+          <DeleteFreeAccountProvider>
```

Comment:
In addition to && `account.type === 'free'` please add `user.orgCount === 1` to ensure that this overlay can only be reached if there is only one org in the free account.